transcribe/scribe_modules/lspci.py

```diff
@@ -0,0 +1,38 @@
+from . import ScribeModuleBaseClass
+
+import sys
+
+
+class Lspci(ScribeModuleBaseClass):
+
+    def __init__(self, input_dict=None, module_name=None, host_name=None,
+                 input_type=None, scribe_uuid=None):
+        ScribeModuleBaseClass.__init__(self, module_name=module_name,
+                                       input_dict=input_dict,
+                                       host_name=host_name,
+                                       input_type=input_type,
+                                       scribe_uuid=scribe_uuid)
+
+    def __iter__(self):
+        for attr, value in self.__dict__.items():
+            yield attr, value
+
+    def parse(self):
+        lspci_data = self._input_dict
+        lspci_lines = lspci_data.split('\n\n')
+
+        if len(lspci_lines) <= 1:
```

Comment:
please dont use this, use the function call ` validate_length(len(lspci_lines), self.module)`